query/adminconsentrequestpolicy/azuread_admin_consent_workflow_enabled.sql

```diff
@@ -0,0 +1,15 @@
+select
+  -- Required Columns
+  id as resource,
+  case
+    when is_enabled then 'ok'
+    else 'alarm'
+  end as status,
+  case
+    when is_enabled then 'Admin consent workflow is enabled.'
```

Comment:
```suggestion
    when is_enabled then tenant_id || ' admin consent workflow is enabled.'
```

I think for all queries, we should always make sure we have a unique identifier starting each `reason`, otherwise users would see multiple lines of `Admin consent workflow is enabled.` (like in the current screenshots). In this case, it's difficult to tell which row is for which tenant, unless they look all the to the right of the screen at the additional dimensions.